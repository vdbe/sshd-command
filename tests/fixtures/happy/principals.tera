---
sshd_command:
    version: 0.2.0
    command: principals
    tokens: '%U %u'
    hostname: false
    complete_user: false
hostname: hostname
search_domains:
    - home.arpa
    - local
---
{% macro principals(fqdn) -%}
{{ fqdn }}
{{ user.name }}@{{ fqdn }}
{%- endmacro input -%}

{{- self::principals(fqdn=hostname) }}
{% for search_domain in search_domains  %}
{{- self::principals(fqdn=hostname ~ "." ~ search_domain) }}
{%  endfor -%}
