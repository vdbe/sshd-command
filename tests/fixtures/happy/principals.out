# Arguments: 1000 user

hostname
user@hostname
hostname.home.arpa
user@hostname.home.arpa
hostname.local
user@hostname.local
